plenum/server/client_authn.py

```diff
@@ -94,9 +94,9 @@ def authenticate(self,
                 raise InvalidSignatureFormat from ex
             ser = self.serializeForSig(msg, topLevelKeysToIgnore=[f.SIG.nm])
             verkey = self.getVerkey(identifier)
-            #
-            # if verkey is None:
-            #     raise CouldNotAuthenticate
+
+            if verkey is None:
```

Comment:
How does it solve the problem in the ticket?
The ticket says that we should check a ledger on whether the verkey is there.